### PR TITLE
Basic structure for development

### PR DESCRIPTION
**Pull request by eeche**
### Description

- Set up the basic structure for development
- Performed a simple network tracepoint monitoring test

### Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Refactor (code improvement that does not add functionality)
- [ ] Documentation update (README, changelog, or other documentation)
- [ ] Other (please explain):

### Checklist

Please check all relevant options.

- [x] I have read the [CONTRIBUTING.md](./CONTRIBUTING.md) document
- [x] My code follows the code style of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings or errors

### CODE REVIEW

File: src/tracepoint/network.c [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+
+            printf("Enter action to block or allow (e.g., block, allow, logging): ");
+            if (fgets(action_str, sizeof(action_str), stdin) == NULL) {
+                break;
+            }
+            action_str[strcspn(action_str, "\n")] = 0;
+
+            if (strcmp(action_str, "allow") == 0) action = 0;
+            if (strcmp(action_str, "block") == 0) action = 1;
+            if (strcmp(action_str, "logging") == 0) action = 2;
```
Comment (harksu) [inline src/tracepoint/network.c]:
```suggestion
/*define at top for constant number*/

define ALLOW_ACTION(example name) 0
define BLOCK_ACTION(example name) 1 
...etc

            if (strcmp(action_str, "allow") == 0) action = 0;
            else if (strcmp(action_str, "block") == 0) action = 1;
            else if (strcmp(action_str, "logging") == 0) action = 2;
```

Comment (harksu) [reply src/tracepoint/network.c]: 상수는 define으로 상위에서 선언하고, 성능적인 이슈를 위해서 케이스를 분기할 땐 불필요한 조건에 대한 체크는 생략해도 될 것 같습니다 :)

File: src/tracepoint/network.c [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+            //         }
+            //         iter_key = next_key;
+            //     }
+            //     printf("Action changed or new policy created. All previous policies for this event_id have been cleared.\n");
+            // }
+            // err = bpf_map__update_elem(skel->maps.event_mode_map, &event_id, sizeof(event_id), &action, sizeof(action), BPF_ANY);
+            // if (err) {
+            //     fprintf(stderr, "Failed to update map: %d\n", err);
+            //     continue;
+            // }
```
Comment (harksu) [inline src/tracepoint/network.c:315]:
```suggestion
```

Comment (harksu) [reply src/tracepoint/network.c:315]: 보통 커밋 내용에는 주석은 포함하지 않는것이 깔끔합니다 :)

**Review by harksu — CHANGES_REQUESTED** [review]
기초 구상인만큼, 기본적인 자잘한 것들만 상의하에 반려하였습니다:)

**Review by harksu — APPROVED** [review]
case 통일 나중에 해주면 좋을 것 같습니다 👍

**Review by KoYejune0302 — APPROVED** [review]
LGTM!

**Pull request merged**